src/main/java/com/devcycle/sdk/server/local/managers/EnvironmentConfigManager.java

```diff
@@ -143,6 +148,12 @@ private ProjectConfig getConfigResponse(Call<ProjectConfig> call) throws DevCycl
                 }
             }
             this.configETag = currentETag;
+            this.configLastModified = lastModified;
+            try {
+                this.eventQueueManager.queueSDKConfigEvent(call.request(), response);
```

Comment:
I think I remember the reason I had to avoid tying these things together being circular dependency issues. It's probably not a big issue here, but might it be better to try not to tie these two main managers together?

Reply:
there's no circular dep issues here; they're distinct and don't relate - tying them together just is a better/cleaner practice than creating a confusing backchannel. 

it's the same structure as c# and Go